extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -69,6 +78,12 @@ export class AmpStoryInteractiveSlider extends AmpStoryInteractive {
     this.inputEl_ = null;
     /** @private {!SliderType}  */
     this.sliderType_ = SliderType.PERCENTAGE;
+    /** @private {boolean}  */
+    this.isInteracting_ = false;
+    /** @private {?number} Reference to timeout so we can cancel it if needed. */
+    this.landingAnimationDelayTimeout_ = null;
+    /**  @private {?number} */
```

Comment:
A comment on this variable could be helpful, like:
`Reference to requestAnimationFrame so we can cancel it if needed.`